base/REPL.jl

```diff
@@ -416,7 +416,7 @@ function add_history(hist::REPLHistoryProvider, s)
     push!(hist.history, str)
     hist.history_file === nothing && return
     entry = """
-    # time: $(Libc.strftime("%Y-%m-%d %H:%M:%S %Z", time()))
+    # time: $( string( now() ) )
```

Comment:
The rest of Base does not space out parentheses like this; it's best to follow the surrounding style. In this case, it should be `$(string(now()))`. Similarly for several other changes here.